### DISCUSSION TITLE
Hiding the terminal window on boot

### DISCUSSION THREAD

**Discussion by Arashatora** [nzbgetcom/nzbget#299] [Q&A] [answered]
I've seen online people say -D to hide the terminal but I apparently am not smart enough to figure out where it goes. Could someone please explain it like I'm 5.

Thanks in advance

**Comment by luckedea** [comment]
@Arashatora How do you start NZBGet (autostart or launch manually)? OS - Windows - Which version?

**Reply by Arashatora** [reply]
Windows 11,newest version of Nzbget. Starts on boot

**Reply by luckedea** [reply]
@Arashatora Easiest solution:

Start the app.
Right-click on the tray icon
Preferences
Disable "Show console window"

**Reply by Arashatora** [reply]
I've started and restarted it with the option checked and uncheck, makes no difference

**Reply by luckedea** [reply]
@Arashatora Ok, sorry, actually this is a known bug on Windows 11 https://github.com/nzbgetcom/nzbget/issues/204 - no trivial fix available, we'll look into it a bit later.

**Reply by 1aTa** [reply, marked as answer]
Workaround:

1. Open Windows Terminal and navigate to Settings
2. Set Default terminal application to Windows Console Host
3. Save settings and enjoy no more console window popups :)

**Reply by hmeijdam** [reply]
That worked just fine. Thanks for sharing.
